.github/workflows/terraform.yml

```
@@ -105,7 +105,7 @@ jobs:
           jq -r '.[]' <<< "$MODULES" | while read MODULE; do
```

```suggestion
          jq -r '.[]' <<< "$MODULES" | sort | while read MODULE; do
```

Nothing to do with your change, but it would be better for the output order to be predictable, otherwise the diff might show unnecessary changes in future